projects/core/src/cart/bundle/occ/adapters/default-occ-bundle-config.ts

```diff
@@ -0,0 +1,13 @@
+import { OccConfig } from 'projects/core/src/occ/config/occ-config';
+
+export const defaultOccBundleConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        bundleStart: 'users/${userId}/carts/${cartId}/bundles?fields=DEFAULT',
```

Comment:
We'd like to control the fields configuration 100% in spartacus. Ideally we should avoid `DEFAULT` in the endpoints configurations. Can we replace it with specific fields that are needed? Perhaps no fields are needed in the response?